Rakefile

```diff
@@ -12,8 +12,7 @@ gemspec = Gem::Specification.load(File.expand_path('xorcist.gemspec',  __FILE__)
 if RUBY_ENGINE == 'jruby'
   require 'rake/javaextensiontask'
   Rake::JavaExtensionTask.new('xorcist', gemspec) do |ext|
-    ext.source_version = '1.6'
-    ext.target_version = '1.6'
+    ext.release = '8'
```

Comment:
this will probably break jruby on java < 8, but it's unlikely someone's still using such old versions